optuna/storages/rdb/storage.py

```diff
@@ -41,7 +41,9 @@ class RDBStorage(BaseStorage):
 
     Args:
         url: URL of the storage.
-        connect_args: Arguments that is passed to :func:`sqlalchemy.engine.create_engine`.
+        engine_kwargs:
+           A dictionary of keyword arguments that is passed to
+           :func:`sqlalchemy.engine.create_engine`.
```

Comment:
I think we have three spaces for indentation in this block.
Please add one more space to keep indentation consistent.

Reply:
Oops..., sorry for my careless mistake.

Fixed : https://github.com/pfnet/optuna/pull/425/commits/2aedba436845f7e9f8963cf19611c1e529988f94